src/Tizen.NUI/src/public/Theme/Theme.cs

```diff
@@ -322,6 +343,12 @@ public void Merge(Theme theme)
 
             if (Version == null) Version = theme.Version;
 
+            if (SmallBrokenImageUrl == null) SmallBrokenImageUrl = theme.SmallBrokenImageUrl;
```

Comment:
For those 3 broken images, I don't think you have to check nullity of destination theme. (Id and version are special properties that represents its identity)
Please just overwrite these values.

Reply:
Thanks, I fixed them